### ISSUE TITLE
UI broken

### ISSUE THREAD

**Issue by clecap** [nextcloud/files_fulltextsearch#75]
When opening the search dialogue the field "Filter by extension" is filled with the entry "txt" but a search nevertheless returns files with other extensions, for example "pdf". However when clicking into the data entry field and entering "txt" it no longer returns files with extension "pdf".

It looks like the initial "txt" is just there to suggest to the user what he might enter and only after entering a value this value really is used.

However, there is no visual attribute which distinguishes these two cases.

Therefore, the situation is very confusing, since the user has no real chance to know which data are really used.

**Comment by daita** [comment]
yes, the 'txt' is a placeholder; i will see if I can make it _less visible_

**Comment by clecap** [comment]
That's cool. and thanx for the quick reply!

It would also be very helpful if this could be done in **all** UI elements with placeholders, in particular in the setting field. I was fighting for an hour against the code until I understood that what I recognized as a default/pre-filled-in field in "settings" with value "my_index" rather was such a placeholder leading to an **empty** setting, which subsequently produced an incomprehensible error message of a misconfigured elasticsearch configuration... Maybe a suggestion would also be to use a different prompting value such as "enter extension, eg txt" instead of a value.
